\documentclass[12pt,a4paper]{article}

\usepackage[top=2cm, bottom=2cm, left=3.5cm, right=3.5cm]{geometry}

\begin{document}

\newcommand{\one}[1]{\vspace{\baselineskip}\hspace{-\parindent}{\large \bf #1}}
\newcommand{\two}[1]{\vspace{\baselineskip}\hspace{-\parindent}{\it #1}}
\newcommand{\square}{\vspace{1.5ex}\noindent}

\thispagestyle{empty}

\one{Research Statement -- Jonathan Dushoff}

\square My primary research interest is using mathematical and computational methods to better understand the evolution, spread and health burden of infectious diseases of humans.  My work combines a variety of approaches, including population biology, population genetics, genomics, immunological modeling and statistical analysis of epidemiological data.

I am interested in infectious disease for both practical and theoretical reasons.  The burden of infectious disease world-wide, particularly on poor countries and communities, is far larger than it should be, and some of my work is aimed at directly or indirectly influencing policy decisions and control strategies.  Additionally, I believe that infectious diseases provide excellent model systems for a variety of evolutionary and population biology questions.

My research uses a variety of mathematical and computational methods to link disease processes across scales, building from within-individual immunological interactions and mutation events, to between-individual infection patterns, to population-level patterns of disease spread and evolution.  My work is highly collaborative, and I have worked on a broad variety of projects with colleagues from disciplines from math, to physics, to ecology to molecular biology.  Mathematically, I am particularly interested in insights from analytic approximations to complex dynamical models, and in novel statistical methods that make use of computational patterns to relax traditional statistical assumptions. Scientifically, my current main foci of interest are influenza, rabies and HIV.

My interests in influenza range from molecular evolution, to classic epidemiology, to statistics.  In the last few years, a huge number of influenza full-genome sequences have become available.  This rich data set allows patterns of evolution to be studied in new ways, and demands new techniques for doing so.  For example, collaborators and I have found evidence for non-random trends in nucleotide composition as influenza viruses evolve, and evidence that patterns of substitution cannot be explained using the traditional paradigm of positively and negatively selected sites.  We are currently investigating mechanistic hypotheses for observed patterns of nucleotide change.  On the epidemiology front, collaborators and I have explored new ways of make use of influenza mortality time series, for example: by harvesting additional information from publicly available death records; developing statistical analysis methods robust to seasonal confounding; and currently by developing new techniques to infer morbidity series from mortality data, and to infer disease parameters from morbidity data.  Finally, collaborators and I continue to construct deterministic and stochastic dynamical models which illuminate the interpretation of incidence patterns.  We have several papers in preparation, review or revision for publication where we study patterns of influenza spread in the light of climatic data, covering both the 1918 and 2009 pandemics, as well as ``annual" flu epidemics from Canada.

I am very fortunate to be associated with a very exciting research project, led by Katie Hampson, which has brought the serious public health problem of human mortality caused by canine rabies back into the focus of mainstream science.  I believe that the contributions that I made to this project in terms of dynamical modeling and statistical methods have been both practically important to unravelling patterns in the data, and also theoretically interesting.  In particular, collaborators and I developed new, permutation-based methods for robustly analyzing rabies incidence time series, created deterministic and stochastic dynamical models of rabies spread on different scales, and used a variety of statistical techniques to robustly analyze detailed rabies-transmission data.

My current interest in HIV research is a natural outgrowth of both my dynamical modeling of influenza and rabies, and my recent work on social norms and determinants of human behavior.  Inspired by the idea that male circumcision may be a useful anti-HIV intervention, and a variety of gender and equity concerns that go along with that idea, members of my lab, collaborators and I have embarked on modeling both the effects of various types of behavior change on the spread of HIV, and in modeling the concomitant spread of behavior change itself.

I plan to continue investigating pathogen biology across scales, from gene-sequence evolution and protein shape and function, through pathogenesis and immunology at the level of individual hosts, to community- and population-level patterns of transmission, pathogen evolution and disease burden.  My overarching goal is to use quantitative techniques to investigate basic and applied biological questions, and to bring this research to bear on practical questions relating to the control of infectious diseases of humans.

I also hope to continue being a broadly collaborative scientist, working with friends and colleagues to bring quantitative methods to bear on interesting biological problems.  Recently, I am actively involved in a variety of collaborations including ecological analysis of metagenomic data, phage-bacteria interactions, dynamics of tumor growth and control, bee ecology and pollination biology, evolutionary advantages of sexual reproduction, and evolution of paralogous genes after genome duplication events in clawed frogs.

\end{document}
